server/src/services/components.service.ts

```diff
@@ -259,3 +280,9 @@ export class ComponentsService {
     return componentWithLayout;
   }
 }
+
+function resolveGridPositionForComponent(dimension: number, type: string) {
```

Comment:
Sane function is declared here
https://github.com/ToolJet/ToolJet/pull/8379/files#diff-b5e1de6ff3c19c9d79fa87a3f5ddf3045631a1d3a7c6c375213175e06a6f02afR66
Shall we move it to some util file and reuse?